packages/plugin-buidler/src/scripts/migrate-legacy-project.ts

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node
+import { migrateLegacyProject } from '@openzeppelin/upgrades-core';
+
+migrateLegacyProject();
```

Comment:
This is an async call so we need to catch a potential rejection, otherwise node fails with `UnhandledPromiseRejection`.

```suggestion
migrateLegacyProject().catch(e => {
  console.error(e);
  process.exit(1);
});
```